### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by johnbdesanto**

### CODE REVIEW

File: docs/config_yaml.md [hunk 1]
```diff
@@ -2,60 +2,121 @@
 
 The GNSS-Acoustic positioning performed by GNATSS is governed by a configuration
 file named "config.yaml". This is a text file with a YAML format that defines
```
Comment (lsetiawan) [inline docs/config_yaml.md:4]: This file name is now not a requirement. So probably just say something to the effect of just a YAML configuration file.

File: docs/input.md [hunk 2]
```diff
@@ -2,32 +2,66 @@
...
+processing modes in sequence or individually. The posfilter module outputs
+GNSS-Acosutic travel times and transducer positions in the community standard
+GNSS-Acoustic data format, so if you have obtained data already in that format
+you may skip the posfilter mode and immediately run the solver.
 
-Step 2: Create working directory and config.yaml file
+The basic step-by-step guide to position a GNSS-Acoustic array using GNATSS is:
 
-Step 3: From working directory, execute
+Step 1: Gather the required input files
```
Comment (lsetiawan) [inline docs/input.md:14]:
```suggestion
Step 1: Gather the [required input files](#required-input-data)
```

Comment (lsetiawan) [reply docs/input.md:14]: This should link to the section below for the required input data

File: docs/input.md [hunk 3]
```diff
@@ -2,32 +2,66 @@
...
+
+Step 2: Create working directory and configuration file
```
Comment (lsetiawan) [inline docs/input.md:16]: Maybe add link to the configuration file page/section for the words `configuration file`

File: docs/position_inversion.md [hunk 4]
```diff
@@ -9,20 +9,27 @@ the Seafloor Geodesy Community experiment.
...
+gnatss run --distance-limit 150 NDP1_2022.yaml
 ```
 
-After a successful run, GNATSS will generate an _output_ folder with the
-following files:
+After a successful run, GNATSS will generate an _output_ directory defined in
+the configuration file, by default _./output_. The output directory will contain
+the following files:
 
+- _gps_solution.csv_: Input data file in the Community Standard GNSS-A format.
```
Comment (lsetiawan) [inline docs/position_inversion.md:31]:
```suggestion
- `gps_solution.csv`: Input data file in the Community Standard GNSS-A format.
```

Change these to be small code stylings to stand out more.

File: docs/config_yaml.md (hunk 1 above)
Comment (johnbdesanto) [reply docs/config_yaml.md]: Done.

File: docs/input.md (hunk 2 above)
Comment (johnbdesanto) [reply docs/input.md]: Inserted link.

File: docs/position_inversion.md (hunk 4 above)
Comment (johnbdesanto) [reply docs/position_inversion.md]: Done.

File: docs/input.md (hunk 3 above)
Comment (johnbdesanto) [reply docs/input.md]: Done.

**Review by lsetiawan — APPROVED** [review]

**Comment by lsetiawan** [comment]
Oh I see. Oops sorry if I merged this prematurely. Please create another PR for additional fixes. Thanks

**Comment by johnbdesanto** [comment]
The issue is a little more complex with that. I can't currently submit a new PR because I can't pass all of the checks required to make a new commit. Here is the context:

I wrote the code `$$ Y*{(k|k-1)} = \Phi*{(k|k-1)} \cdot Y_{(k-1|k-1)} $$` in markdown, which should be transcribed to

$$ Y_{(k|k-1)} = \Phi_{(k|k-1)} \cdot Y_{(k-1|k-1)} $$

The pre-commit check fails and re-writes this as `$$ Y*{(k|k-1)} = \Phi*{(k|k-1)} \cdot Y\_{(k-1|k-1)} $$`, which renders as

$$ Y*{(k|k-1)} = \Phi*{(k|k-1)} \cdot Y\_{(k-1|k-1)} $$

So I need a way to overwrite the pre-commit bot that is erroneous screwing with the markdown.

**Pull request merged**